uiautomator2/__init__.py

```diff
@@ -164,6 +164,24 @@ def connect_usb(serial=None):
     return d
 
 
+def connect_url(addr=None):
+    """
+        Args:
+            addr (str) uiautomator server address.
+
+        Returns:
+            UIAutomatorServer
+
+        Examples:
+            connect_url("10.0.0.1/atx2/")
+    """
+    if '://' not in addr:
+        addr = 'http://' + addr
+    u = urlparse.urlparse(addr)
```

Comment:
The addr need to rstrip("/")

Reply:
when url join:
>>> urlparse.urljoin("http://a.b.c/e/f","/kkk")
'http://a.b.c/kkk'
>>> urlparse.urljoin("http://a.b.c/e/f/","/kkk")
'http://a.b.c/kkk'
>>> urlparse.urljoin("http://a.b.c/e/f","kkk")
'http://a.b.c/e/kkk'   (this missing 'f' )
>>> urlparse.urljoin("http://a.b.c/e/f/","kkk")
'http://a.b.c/e/f/kkk'  （this is needed, path2url() has modified too ）